.kokoro/presubmit/node12/samples-test.cfg

```diff
@@ -8,5 +8,5 @@ env_vars: {
 
 env_vars: {
   key: "SECRET_MANAGER_KEYS"
-  value: "nodejs-ai-platform-env"
+  value: "nodejs-ai-platform-env, ucaip_samples_secrets"
```

Comment:
we'll need to add ` .kokoro/presubmit/node12/samples-test.cfg` to excludes, in `synth.py`, so that it's not overwritten when the templates update nightly.

Reply:
Done.